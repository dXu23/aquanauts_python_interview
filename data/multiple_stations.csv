Station Name,Measurement Timestamp,Air Temperature
Oak Street Weather Station,01/01/2016 04:00:00 PM,-1
Foster Weather Station,01/01/2016 06:00:00 AM,-6.39
Oak Street Weather Station,01/01/2016 03:00:00 PM,-1.1
Foster Weather Station,01/01/2016 05:00:00 PM,-1.67
Oak Street Weather Station,01/01/2016 02:00:00 PM,-1.3
Foster Weather Station,01/01/2016 09:00:00 AM,-4.39
Oak Street Weather Station,01/01/2016 01:00:00 PM,-1.9
Foster Weather Station,01/01/2016 08:00:00 AM,-5.06
Oak Street Weather Station,01/01/2016 12:00:00 PM,-2.2
