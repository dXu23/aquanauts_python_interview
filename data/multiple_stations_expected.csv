Station Name,Date,Min Temp,Max Temp,First Temp,Last Temp
Oak Street Weather Station,01/01/2016,-2.2,-1.0,-2.2,-1.0
Foster Weather Station,01/01/2016,-6.39,-1.67,-6.39,-1.67
